types/bootstrap/test/global-script.ts

```diff
@@ -0,0 +1,4 @@
+// test global script usage
+
+const modal = new bootstrap.Modal("#myModal");
```

Comment:
You might want to add a test for a global constructor that takes an `Element` as the first argument. That would add completeness.

Reply:
done, amended, thx!